### ISSUE TITLE
Themes preview won't load

### ISSUE THREAD

**Issue by dpinna13** [modufolio/Koken-App#26]
Hi there, I recently updated my koken instance and thank to this I was able to extend its precious life! Thank you for doing this incredible work!

I tested multiple times and I wasn't able to select a new theme (preview) and the switch to a new theme. 

Two screenshots from two different instances:
![image](https://github.com/user-attachments/assets/79211ecb-3c18-4f10-99c1-da5bd7cce6f9)
![image](https://github.com/user-attachments/assets/db9bc2f3-3bbe-43be-bd09-77890987f51f)

1. Preview images won't load, and the spinner remains
2. Cannot interact on any installed theme, so clicking on a theme won't load the preview panel, and therefore can't switch theme. 
3. I was looking within the code to see if the theme switch can be promted by url `.../koken-dev/admin/#/site/themes/preview:elementary` but I wasn't successful either. 

Additional details:
No plugins enabled. 
Version 1.2.0
PHP 8.1.29

**Comment by aemsx** [comment]
try to remove the boulevard theme manually from your installation. afair there are some issues with this theme that also might prevent your overview from loading correctly.

no forther issues here with php 8.2.20 on version 1.1.9

**Comment by dpinna13** [comment]
Yeah, I read that conversation, and even in the instance without boulevard I still can't get to switch themes.  
![image](https://github.com/user-attachments/assets/c316f094-e005-470c-9cd9-fd9b4360464d)

I also tried removing all 'drafts' from draft history, leaving the current one only.

![image](https://github.com/user-attachments/assets/2c155135-0a1b-4481-a5ed-c2c65eb69bdf)

**Comment by aemsx** [comment]
just updated to 1.2.0 - still no issues. got Elementary theme in v1.7.2 (+ others). maybe you want to check out https://github.com/korbendallas01?tab=repositories for some updates on your theme(s)

**Comment by dpinna13** [comment]
Thank you for the link. 

Can you suggest the best way to get more details on this issue? Where can I retrieve/generate the logs that might provide some clues?
